src/aioswitcher/device/tok.py

```diff
@@ -0,0 +1,35 @@
+# Copyright Tomer Figenblat.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# flake8: noqa
+
+"""Switcher convert token to valid packet string. Input: Token (String), Output: Packet value (String)"""
+
+from base64 import b64decode
+from binascii import hexlify
+from typing import Union
+
+from Crypto.Cipher import AES
+from Crypto.Util.Padding import unpad
+
+secret_key = b"jzNrAOjc%lpg3pVr5cF!5Le06ZgOdWuJ"
+
+
+def he(token: Union[str, None] = None) -> str:
```

Comment:
```suggestion
def convert_token_to_packet_data(token: Union[str, None] = None) -> str:
```
or something similar

Reply:
I would also consider just adding these lines inside `convert_token_to_packet` since it is used only there and only few lines

Reply:
Looks good I will refactor

Reply:
> I would also consider just adding these lines inside `convert_token_to_packet` since it is used only there and only few lines

Sorry but I didn't understand what you mean..

Reply:
It is done I put the tok.py logic under `convert_token_to_packet`